### DISCUSSION TITLE
How to get original db field name in a generator?

### DISCUSSION THREAD

**Discussion by remorses** [prisma/prisma#13598] [Q&A]
I want to get the original db name inside a generator but i cannot find a way to do it

I mean a model field name defined with `@map`

**Comment by austincrim** [comment]
Hey @remorses 👋,

Mapped field information is not currently available in the DMMF that is used by generators.

Please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with some details about your use case and what you need, thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
